### PR TITLE
added naming conventions examples

### PR DESCRIPTION
**Pull request by AlessandroMinoccheri**

### CODE REVIEW

File: examples/naming-conventions.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Naming Conventions
+
+In this example, we would like to share some simple rules to apply some naming conventions in your project.
+
+Naming conventions can help the team to create and rename new files with a common shared standard.
```
Comment (micheleorselli) [inline examples/naming-conventions.md:5]: classes, not files

File: examples/naming-conventions.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Naming Conventions
+
+In this example, we would like to share some simple rules to apply some naming conventions in your project.
+
+Naming conventions can help the team to create and rename new files with a common shared standard.   
+
+In the following rules, written into `phparkitect.php` file into the root of a project, we said that:
```
Comment (micheleorselli) [inline examples/naming-conventions.md:7]: That would be the case for every example, should we repeat it every time? 🤔

File: examples/naming-conventions.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+```
+* All files inside namespace App\Infrastructure\Controller should end with Controller
+* All files inside namespace App\Domain\Auction\Bids should start with Bid
```
Comment (micheleorselli) [inline examples/naming-conventions.md:10]: classes, not files

**Review by micheleorselli — CHANGES_REQUESTED** [review]

File: examples/naming-conventions.md (hunk 3 above)
Comment (AlessandroMinoccheri) [reply examples/naming-conventions.md:10]: Right

File: examples/naming-conventions.md (hunk 1 above)
Comment (AlessandroMinoccheri) [reply examples/naming-conventions.md:5]: Right.

File: examples/naming-conventions.md (hunk 2 above)
Comment (AlessandroMinoccheri) [reply examples/naming-conventions.md:7]: ok, do you think to remove all the seven line? 
Do you have some suggestions @micheleorselli ?

Comment (Jean85) [reply examples/naming-conventions.md:7]: What about:
```
In this example, the desired rules are:
# ...
Which can be translated in the configuration like this:
```

Comment (AlessandroMinoccheri) [reply examples/naming-conventions.md:7]: Sounds good to me! 👍 
@micheleorselli ?

Comment (micheleorselli) [reply examples/naming-conventions.md:7]: yup, good to me!

**Pull request merged**